### PR TITLE
Add draft story of the project

### LINKED ISSUE #27: Story of how the project came to be
Write a short paragraph on how the project came to be - what inspired it, how did it happen.

### PR DESCRIPTION
**Pull request by mbonsma**
Resolves #27.

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -76,4 +76,10 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
 
 # Main Body
 
+## Story of the project
+
+We are a group of graduate students who connected through the [UofT Coders](http://uoftcoders.github.io/) club at the University of Toronto. We created and ran a fourth-year course in the Department of Ecology and Evolutionary Biology titled “Theoretical Ecology and Reproducible Quantitative Methods in R.” We modelled the structure and portions of the course content on “Reproducible Quantitative Methods” a course created by Dr. Christie Bahlai [ ], modifying the lesson content to include more theoretical ecology topics but maintaining the focus on reproducibility and computational skills. Our teaching team was six graduate students from several different fields (Physiology, Biomedical Engineering, Physics, Psychology, and Nutritional Science); we divided course topics among each instructor to develop and deliver individual lessons and assignments.
```
Comment (lwjohnst86) [inline paper.md]: club -> group

Comment (lwjohnst86) [reply paper.md]: `[“Reproducible Quantitative Methods”](https://cbahlai.github.io/rqm-template/)`

Comment (lwjohnst86) [reply paper.md]: It might be nice to say *why* we started the course, but that might be too much words already.

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

File: paper.md (hunk 1 above)
Comment (mbonsma) [reply paper.md]: Yeah, good point - it could be a short thing, like we saw a need for more quantitative skills for biology students.

Comment (joelostblom) [reply paper.md]: I can add a background motivation and some changes within the next hour. @lwjohnst86 may I repurpose from the rostools application?

Comment (joelostblom) [reply paper.md]: @lwjohnst86 Nvm, I am not, but almost done

Comment (joelostblom) [reply paper.md]: @mbonsma I added a PR on your branch with a new paragraph and Luke's comments. Maybe it's too long now... =/ Is there are word limit?

Comment (mbonsma) [reply paper.md]: The whole thing should be about 1000 words, but we can do the trimming at the end.

**Review by joelostblom — APPROVED** [review]

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**